src/Vts/MonteCarlo/DataStructuresValidation/SimulationInputValidation.cs

```diff
@@ -61,10 +61,16 @@ private static ValidationResult ValidateSourceInput(ISourceInput sourceInput, IT
                     "Source input not valid given tissue definition",
                     "Alter sourceInput.InitialTissueRegionIndex to be consistent with tissue definition");
             }
+            if (sourceInput is FluorescenceEmissionAOfRhoAndZSourceInput)
+                return FluorescenceEmissionAOfRhoAndZSourceInputValidation.ValidateInput(sourceInput);

```

Comment:
Did we decide that if we inline the return then we wouldn't need curly braces, this is not exactly inline, it's on the next line which could lead to bugs if somebody forgets the braces. I know these lines are long but should we inline these?

Reply:
I decided to use curly braces instead of inline because to be consistent with code above it which could be inline but uses curly braces.